tools/jbrowse/jbrowse_to_container.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
+<tool id="jbrowse_to_container" name="Add organisms to JBrowse container" version="@WRAPPER_VERSION@">
+    <description>will replace any existing organism</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="aggressive"><![CDATA[
+
+        if [ -z "@DATA_DIR@" ]; then
+            echo '@DATA_DIR@ variable must be set to use this tool';
+            exit 1;
+        fi
+
+        &&
+
+        rm -rf "@DATA_DIR@/"*
+
+        &&
+
+        #for $org in $organisms:
+            #if $org.advanced.unique_id:
+                #set dataset_name = $org.advanced.unique_id
+            #else:
+                #set dataset_name = $org.jbrowse.id
+            #end if
+
+            cp -R '${org.jbrowse.extra_files_path}/data/' "@DATA_DIR@/${dataset_name}"
```

Comment:
Note to self: If the rmrf above was meant to be `@DATA_DIR@/${dataset_name}` then this statement is fine, otherwise will copy into dir (e.g. org/trackList.json (old) + org/org/trackList.json (new)) which isn't what is wanted.